Example/TableViewModelProvider.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlanGrid
+//  https://www.plangrid.com
+//  https://medium.com/plangrid-technology
+//
+//  Documentation
+//  https://plangrid.github.io/ReactiveLists
+//
+//  GitHub
+//  https://github.com/plangrid/ReactiveLists
+//
+//  License
+//  Copyright © 2018-present PlanGrid, Inc.
+//  Released under an MIT license: https://opensource.org/licenses/MIT
+//
+
+import Foundation
+import ReactiveLists
+
+extension TableViewController {
+
+    /// Pure function mapping new state to a new `FluxTableViewModel`.  This is invoked each time the state updates
+    /// in order for ReactiveLists to update the UI.
+    static func viewModel(forState groups: [UserGroup], onDeleteClosure: @escaping (User) -> Void) -> FluxTableViewModel {
+        let sections = groups.map { group in
+            FluxTableViewModel.SectionModel(
+                headerTitle: group.name,
+                headerHeight: 20,
+                cellViewModels: group.users.map { UserCell(user: $0, onDeleteClosure: onDeleteClosure) },
+                diffingKey: group.name
+            )
+        }
+        return FluxTableViewModel(sectionModels: sections)
+    }
+}
+
+
```

Comment:
I think it'd be good to put this in the controller class directly (same file), now that the boilerplate is removed.

I think that will make it easier -- if you can see all the pieces at once 😄

Reply:
Agree that that's better (at least for the demo projects).

Reply:
Cool